memeolist/app/src/main/assets/mobile-services.json

```diff
@@ -10,6 +10,7 @@
       "type": "sync",
       "url": "https://api.graph.cool/simple/v1/cjiyvc1wa40kg011846ev0ff8",
       "config": {
+        "subscription": "wss://subscriptions.us-west-2.graph.cool/v1/cjiyvc1wa40kg011846ev0ff8"
```

Comment:
[Question] Do we need separate subscription url. When websockets are used they can be part of the second URL. On IOS you can only use single url at the time. Android api seems to be the same.

Reply:
Yes, in theory, we can have only one endpoint/URL. I have added it separately because in my tests I was using [Graph Cool](http://graph.cool) and it uses a separate one for subscription/WebSocket